scripts/spcgeonode/docker-compose.yml

```diff
@@ -153,19 +169,22 @@ services:
 
   # PostGIS database.
   postgres:
-    image: mdillon/postgis:9.6-alpine
+    image: geonode/postgis:10
     environment:
       - POSTGRES_PASSWORD=${POSTGRES_PASSWORD}
+      - GEONODE_DATABASE=${GEONODE_DATABASE}
+      - GEONODE_DATABASE_PASSWORD=${GEONODE_DATABASE_PASSWORD}
+      - GEONODE_GEODATABASE=${GEONODE_GEODATABASE}
+      - GEONODE_GEODATABASE_PASSWORD=${GEONODE_GEODATABASE_PASSWORD}
+      - DATABASE_URL=postgres://${GEONODE_DATABASE}:${GEONODE_DATABASE_PASSWORD}@postgres:5432/${GEONODE_DATABASE}
+      - GEODATABASE_URL=postgis://${GEONODE_GEODATABASE}:${GEONODE_GEODATABASE_PASSWORD}@postgres:5432/${GEONODE_GEODATABASE}

```

Comment:
You could use `x-common-db` similarly to `x-common-django` to avoid duplicated variables.

Reply:
good idea. Will enhance the setup with more commons and yml anchors.

Reply:
Two anchors are not working as discussed on gitter on [April 15, 2019 4:44 PM](https://gitter.im/GeoNode/general?at=5cb498c793fb4a7dc233185b). Which is a pitty.

The proposed setup has only one point of configuration: the global .env file

All changes should be done in there and will be ingested into the appropriate places via docker-compose and shell scripts.